turte_graphics/turtle.js

```diff
@@ -0,0 +1,111 @@
+class Turtle {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+    this.position = [x, y];
+    this.path = [[x, y]];
+    this.direction = "x";
+  }
+
+  forward(steps) {
+    for (let i = 0; i < steps; i++) {
+      if (this.direction === "x") {
+        this.x++;
+      } else if (this.direction === "-y") {
+          this.y--;
+      } else if (this.direction === "-x") {
+        this.x--;
+      } else {
+          this.y++;
+      }
```

Comment:
Often when you have a series of `if`/`else if`, it makes things much cleaner if you use a [`switch` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) instead.

Reply:
done!